src/test/groovy/org/gradle/wrapperupgrade/GradleWrapperUpgradePluginFuncTest.groovy

```diff
@@ -115,11 +150,17 @@ wrapperUpgrade {
         result.output.contains('Configuration cache entry stored.')
 
         when:
+        GradleRunner.create()
+            .withProjectDir(testProjectDir)
+            .withPluginClasspath()
+            .withGradleVersion(determineGradleVersion().version)
+            .withArguments('clean')
+            .build()
         result = GradleRunner.create()
             .withProjectDir(testProjectDir)
             .withPluginClasspath()
             .withGradleVersion(determineGradleVersion().version)
-            .withArguments('clean', 'upgradeGradleWrapperAll', '--configuration-cache', '-DwrapperUpgrade.dryRun', '-DwrapperUpgrade.unsignedCommits', '--stacktrace')
+            .withArguments('upgradeGradleWrapperAll', '--configuration-cache', '-DwrapperUpgrade.dryRun', '-DwrapperUpgrade.unsignedCommits')
```

Comment:
this change revealed a weird behavior with CC: the `clean` task was executed after the `upgradeGradleWrapperAll`, not sure why. By executing it first separately, we make sure the clones projects are cleaned up.

Reply:
Oh, wow I thought `clean` would be handled specially.